app/models/host/discovered.rb

```diff
@@ -178,4 +182,10 @@ def compute_resource
     false
   end
 
+  def lookup_value_match
```

Comment:
might this be an issue in core?  If Nic::Managed is used on Host::Base and upwards then perhaps lookup_value_match should move from Host::Managed to Host::Base.